charts/jaeger/templates/allinone-deploy.yaml

```diff
@@ -123,6 +136,11 @@ spec:
         {{- toYaml .Values.allInOne.podSecurityContext | nindent 8 }}
       serviceAccountName: {{ template "jaeger.fullname" . }}
       volumes:
+    {{- if .Values.v2.enabled }}
+        - name: jaeger-v2-config
+          configMap:
+            name: {{ include "jaeger-v2.fullname" . }}
```

Comment:
where is `jaeger-v2.fullname` coming from? does config map name need to be parameterized?

Reply:
it is define in _helper.tql in line 734

Reply:
why does the name of config map need to be parameterized?